apps/builder/app/canvas/shared/inert.ts

```diff
@@ -31,3 +43,10 @@ const setAutoDisposeInert = (timeout: number) => {
  */
 export const setInert = () => setAutoDisposeInert(AUTO_DISPOSE_INERT_TIMEOUT);
 export const resetInert = () => setAutoDisposeInert(DISPOSE_INERT_TIMEOUT);
+
+// window.self !== window.top means we are on canvas
+if (typeof window !== "undefined" && window.self !== window.top) {
+  window.addEventListener("pointerdown", () => {
```

Comment:
don't we need to clean up all these even listeners at somepoint? For example hot reload will probably cause issues here and there if there is no init and cleanup

Reply:
Somehow we need, but everytime we have tried to use module.hot it became much more worse. 
Current code should not do anything bad even in case of hot reload imo